src/main/java/org/roda_project/commons_ip2/validator/component/descriptiveMetadataComponent/DescriptiveMetadataComponentValidator.java

```diff
@@ -570,6 +571,12 @@ private ReporterDetails validateCSIP24(final StructureValidatorState structureVa
           } else {
             path.append(metsValidatorState.getMetsPath()).append(hrefDecoded);
           }
+          if (!StringUtils.isNotBlank(href)){
```

Comment:
Instead of a double negation, `StringUtils.isBlank(href)` would read better. :)
The same check should also be added to the else branch below when not checking a zip file.